File-Formats/DocIO/word-to-pdf.md

```diff
@@ -12,6 +12,10 @@ documentation: UG
 
 The Word document files are converted as a PDF document with a few lines of code by using the Essential DocIO. It works perfectly when you create an input Word document from scratch or load an existing Word document and easily converted into PDF. 
 
+To quickly start converting a Word document to PDF, please check out this video:
```

Comment:
To quickly start converting a Word document to a PDF, please check out this video: